### ISSUE TITLE
[v2.0.0-rc.0] Broken support for extended fence language

### ISSUE THREAD

**Issue by simonbrunel** [eslint/eslint-plugin-markdown#166]
This is a regression of the feature introduced by a5d0ccec6f14f4808a535c0f06273d1e9892be15. 

For example, using the same input as the unit test:

````
```js more words are ignored
var answer = 6 * 7;
```

```js
var answer = 6 * 7;
```
````

Running ESLint on the markdown above:

```
> eslint *.md


test.md
  6:1  error  Unexpected var, use let or const instead  no-var

✖ 1 problem (1 error, 0 warnings)
```

Should report 2 errors with the following config:

```js
module.exports = {
  extends: 'plugin:markdown/recommended',
  rules: {
    'no-var': 'error'
  }
};
```

I haven't been able to run tests locally (failed to `npm install` this repo on Windows) but I think that the [associated unit test](https://github.com/eslint/eslint-plugin-markdown/blob/d30c50f46237af2fdef0a8a21fb547ed8e6c4d80/tests/lib/processor.js#L271-L280) doesn't guarantee that the returned block will be processed by ESLint. I guess the returned `block.filename` is `'0.js more words are ignored'` which doesn't pass the glob matching on this virtual block filename.

**Comment by btmills** [comment]
> I guess the returned `block.filename` is `'0.js more words are ignored'` which doesn't pass the glob matching on this virtual block filename.

You were right! Thanks so much for the bug report. Fix is in #167.

**Comment by btmills** [comment]
> I haven't been able to run tests locally (failed to `npm install` this repo on Windows)

Whoops. Thanks for reporting that as well. #168 should fix it.
